src/ray/raylet/scheduling/scheduling_policy_test.cc

```diff
@@ -40,6 +41,14 @@ class SchedulingPolicyTest : public ::testing::Test {
   scheduling::NodeID remote_node_2 = scheduling::NodeID(2);
   scheduling::NodeID remote_node_3 = scheduling::NodeID(3);
   absl::flat_hash_map<scheduling::NodeID, Node> nodes;
+
+  SchedulingOptions HybridOptions(
+      float spread, bool avoid_local_node, bool require_node_available,
+      bool avoid_gpu_nodes = RayConfig::instance().scheduler_avoid_gpu_nodes()) {
+    return SchedulingOptions(SchedulingType::HYBRID,
```

Comment:
Use SchedulingOptions::Hybrid() to construct?

Reply:
ah SchedulingOptions::Hybrid() uses the spread_threshold from config but in the test we do want to test different spread_threshold.